recipes/jsonrpcclient/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "jsonrpcclient" %}
+{% set version = "3.3.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f2c5d6bed0d37e4f1f7e76c6e0ae2669ef2facd709508398cc54086b83a6dcf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

I think it could be ``noarch``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

---

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python